Form/Type/BooleanType.php

```diff
@@ -55,12 +56,18 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $choices =  array(
+            self::TYPE_YES  => 'label_type_yes',
+            self::TYPE_NO   => 'label_type_no',
+        );
+
+        if (version_compare(Kernel::VERSION, '3.0', '>=')) {
```

Comment:
how about this check? 
`if ( method_exists(get_parent_class(), 'getBlockPrefix') ) {
            $choices = array_flip($choices);
        }`

Or maybe 

`if ( class_exists('Symfony\Component\Form\Util\StringUtil') ) {
  $choices = array_flip($choices);
}`